src/core/editform/qgsattributeeditorelement.cpp

```diff
@@ -86,6 +119,21 @@ QgsAttributeEditorElement *QgsAttributeEditorElement::create( const QDomElement
     else
       newElement->setShowLabel( true );
 
+    // Label font and color
+    if ( element.hasAttribute( QStringLiteral( "labelColor" ) ) )
+    {
+      newElement->setLabelColor( QgsSymbolLayerUtils::decodeColor( element.attribute( QStringLiteral( "labelColor" ) ) ) );
+    }
+
+    QFont font;
+    QgsFontUtils::setFromXmlChildNode( font, element, QStringLiteral( "labelFont" ) );
+    newElement->setLabelFont( font );
+
+    if ( element.hasAttribute( QStringLiteral( "overrideLabelStyle" ) ) )
+    {
+      newElement->setOverrideLabelStyle( element.attribute( QStringLiteral( "overrideLabelStyle" ) ) == QChar( '1' ) );
```

Comment:
does this work? When storing we are just storing the bool value directly, so I'm not sure that it's guaranteed to be a "0" or "1" character. I think it's safer to be explicit in the write xml and make that `mOverrideLabelStyle ? "1" : "0"`, just in case Qt changes the undocument behaviour in future